development/bot_handler.py

```diff
@@ -0,0 +1,32 @@
+import requests
+from environment import get_env_variable
+
+from auth_app.models import Bot
+
+
+SERVER_URL = get_env_variable('SERVER_URL')
+SERVER_PORT = get_env_variable('SERVER_PORT')
+
+
+def get_data_users():
```

Comment:
NIT: I would rephrase it as `get_users_data`

Reply:
done!